src/components/SplashScreenHider/index.native.tsx

```diff
@@ -2,6 +2,7 @@ import {useCallback, useRef} from 'react';
 import {StyleSheet, ViewStyle} from 'react-native';
 import Reanimated, {Easing, runOnJS, useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
 import Logo from '@assets/images/new-expensify-dark.svg';
+import ImageSVG from '@components/ImageSVG';
```

Comment:
NAB but I prefer `SVGImage` over `ImageSVG`

Reply:
You're right` SVGImage` sounds better but there is one benefit of naming this component `ImageSVG`. This allows to store the Image and ImageSVG components side by side in the `src/components` package, which is quite intuitive :)